### ISSUE TITLE
WEB-DL Quality Not Upgrading

### ISSUE THREAD

**Issue by rflack1717** [SickRage/SickRage#1994]
### Before submitting your issue:

Enable debug logging in SickRage settings, reproduce the error (be sure to disable after the bug is fixed)

Branch: master
Commit: 34d0987400b3d50f550f55472d417c4e5b263f55
Version: 2016.06.14-1
Database Version: 44.0
What you did: Changed desired quality from 720p web-dl to 1080p web-dl then ran backlog search.
What happened: Backlog search ran but it thinks 1080p is less/same as 720p.
What you expected: 1080p to be downloaded and replace the 720p.
Logs:
```
2016-06-21 17:56:16 DEBUG SEARCHQUEUE-BACKLOG-269750 :: [BTN] :: Ignoring result Wayward.Pines.S02E04.Exit.Strategy.1080p.WEB-DL.DD5.1.H.264-ViSUM.
2016-06-21 17:56:16 DEBUG SEARCHQUEUE-BACKLOG-269750 :: [BTN] :: Episode already exists with quality 720p WEB-DL and the found result has same/lower quality, ignoring found result for Wayward Pines S02E04 with quality 1080p WEB-DL
2016-06-21 17:56:16 DEBUG SEARCHQUEUE-BACKLOG-269750 :: [BTN] :: Any,Best = [ 1080p WEB-DL ] [ 1080p BluRay ] Found = [ 1080p WEB-DL ]
2016-06-21 17:56:16 DEBUG SEARCHQUEUE-BACKLOG-269750 :: [BTN] :: Parsed Wayward.Pines.S02E04.Exit.Strategy.1080p.WEB-DL.DD5.1.H.264-ViSUM into Wayward Pines - S02E04 [GROUP: ViSUM] [ABD: False] [ANIME: False] [whichReg: ['standard']]
```

So in short, I changed a series from 720p web-dl to 1080p web-dl and then manually ran a backlog search. I expected the 720p episodes to be replaced by 1080p episodes, but the log above seems to show that SickRage doesn't understand that 1080p is better?

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [x] Post debug logs, either inline (for smaller logs) or using gist
- [x] Please wrap inline logs inside \`\`\` and \`\`\` for readability

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by neoatomic** [comment]
Not sure why its not overwriting. But if you remove the old files than it should work.

Also if you remove the show from sickrage and re-add it, that should clear the old history. that also might work.
